frappe_graphql/utils/loader.py

```diff
@@ -8,20 +8,20 @@
 
 from .exceptions import GraphQLFileSyntaxError
 
-
-graphql_schema = None
+graphql_schemas = None
```

Comment:
@e-lobo since `get_schema` is hit up for every single request, we could do some optimization here
- initialize `graphql_schemas` as {} instead of `None` in Line. 11
- If this is done, we can reduce the if statement in line 17 to just `if frappe.local.site in graphql_schemas` since we know for sure graphql_schemas is not None and is an instance of dict
- We could completely remove the if statement at line 19

Reply:
@fahimalizain done 👍